rubrics/impl/src/test/org/sakaiproject/rubrics/impl/test/RubricsServiceTests.java

```diff
@@ -315,11 +318,23 @@ public void deleteRubric() {
 
         rubricsService.deleteRubric(rubricBean.getId());
 
+        evaluations = evaluationRepository.findAll();
+        assertEquals(0, evaluations.size());
+
+        // TODO: These are commented out because I can't get cascade delete to work on the rubric
```

Comment:
can you put these in an IF statement? 

if (using mysql) {

Reply:
I can look at that. I don't know how you do that in the actual test though without hacking up the repository interface to expose the hibernate session. There may be a way.